src/components/dashboard/sidebar/LearnerCreditSummaryCard.jsx

```diff
@@ -31,32 +32,25 @@ const LearnerCreditSummaryCard = ({
     cardClassNames={className}
   >
     <p data-testid="learner-credit-summary-text">
-      { LEARNER_CREDIT_CARD_SUMMARY }
+      { assignmentOnlyLearner ? LEARNER_CREDIT_ASSIGNMENT_ONLY_SUMMARY : LEARNER_CREDIT_CARD_SUMMARY }
     </p>
 
     {expirationDate && (
       <p data-testid="learner-credit-summary-end-date-text">
         Available until <b>{dayjs(expirationDate).format('MMM D, YYYY')}</b>
       </p>
     )}
-
-    {searchCoursesCta && (
-      <Row className="mt-3 d-flex justify-content-end">
-        <Col xl={12}>{searchCoursesCta}</Col>
-      </Row>
-    )}
```

Comment:
Just to confirm, this is seemingly unused, right? This component never receives the `searchCoursesCta` prop?

Reply:
Correct, the only use case for the prop was in a test. the component was reliant on the parents reference to the `searchCourseCTA`